hyppo/ksample/smoothCF.py

```diff
@@ -0,0 +1,184 @@
+import numpy as np
+from scipy.stats import chi2
+from ._utils import _CheckInputs
+from hyppo.ksample.base import KSampleTest, KSampleTestOutput
+
+
+class SmoothCFTest(KSampleTest):
+    r"""
+    Smooth Characteristic Function test statistic and p-value
+
+    The Smooth Characteristic Function test is a two-sample test that uses
+    differences in the smoothed (analytic) characteristic function of
+    two data distributions in order to determine how different
+    the two data are :footcite:p:`chwialkowski2015fast`.
+
+    Parameters
+    ----------
+    num_randfreq: integer
+        Used to construct random array with size ``(p, q)`` where `p` is the number of
+        dimensions of the data and `q` is the random frequency at which the
+        test is performed. These are the random test points at which test occurs (see notes).
+    random_state: integer
```

Comment:
same comment about `random_state` as before, and `int`

Reply:
changed both